Tests/Form/Type/TranslationTypeTest.php

```diff
@@ -35,8 +35,13 @@ public function testSubmitValidData()
 
         $translation = $this->createTestTranslation($formData);
 
-        $type = new TranslationType();
-        $form = $this->factory->create($type);
+        if (method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')) {
+            $typeName = 'Asm\TranslationLoaderBundle\Form\Type\TranslationType';
+        } else {
+            $typeName = 'asm_translation';
+        }
+
+        $form = $this->factory->create($typeName, new Translation());
```

Comment:
@maschmann Any idea why passing the `Translation` entity is necessary for Symfony 3, but it did work before without this?